### PR TITLE
IDVA5-833 home address changes

### PR DESCRIPTION
**Pull request by sardhani-ch**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by ttingle-ch — APPROVED** [review]
LGTM

File: src/controllers/homeAddressController.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            previousPage,
+            AddressManualLink,
+            currentUrl,
+            payload: req.body,
+            firstName: clientData?.firstName,
+            lastName: clientData?.lastName,
+            ...pageProperties
+        });
+    } else {
+        const postcode = req.body.postCode;
```
Comment (sampankumar) [inline src/controllers/homeAddressController.ts:56]: Please move all of this code to service layer.

**Pull request merged**